src/main/java/wis/util/WisException.java

```diff
@@ -0,0 +1,80 @@
+package wis.util;
+
+import java.io.IOException;
+
+import wis.Action;
+
+public class WisException {
+    public static void ActionExceptionHandler(Action action) {
```

Comment:
Would using a verb for the method name be a better idea? i.e `handleActionException` instead of `ActionExceptionHandler`

Should method names should be in camelCase instead of PascalCase?

Reference: https://nus-cs2103-ay2324s2.github.io/website/schedule/week4/topics.html#w4-6-code-quality-naming

<img width="739" alt="image" src="https://github.com/nus-cs2103-AY2324S2/ip/assets/44436551/36137fe3-4682-4ba8-8e69-4dc997188479">

Reply:
Noted. Thank you!